### PR TITLE
Add DragonCoin (BEP20)

### PR DESCRIPTION
**Pull request by DragonCoin71**
Adding info for BEP20 token 'DragonCoin'.

PR created by <a href="https://assets.trustwallet.com">TW Assets Management</a>

### CODE REVIEW

**Comment by Cryptocool1** [comment]
hi, 
can you please add links to your PR?
example: https://developer.trustwallet.com/add_new_asset#info-json-contents

**Comment by DragonCoin71** [comment]
Hello, 
Should i send links here? Or i should add them one by one in info.json

File: blockchains/smartchain/assets/0x645660bD50F7229F1a975F1231c82EBAa31A74Fe/info.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "name": "DragonCoin",
+    "type": "BEP20",
+    "symbol": "DRON",
+    "decimals": 8,
+    "website": "https://dragon-dron.com/",
+    "description": "DragonCoin is a promising project for all the people of the world. Our goal in creating this project to show the power of the dragon to the whole world. Which means that we will maximize holder investment by DragonCoin project and we will spend part of the profits to use of Solar energy.",
+    "explorer": "https://bscscan.com/token/0x645660bD50F7229F1a975F1231c82EBAa31A74Fe",
+    "status": "active",
+    "id": "0x645660bD50F7229F1a975F1231c82EBAa31A74Fe"
```
Comment (DragonCoin71) [inline blockchains/smartchain/assets/0x645660bD50F7229F1a975F1231c82EBAa31A74Fe/info.json:10]:
```suggestion
    "id": "0x645660bD50F7229F1a975F1231c82EBAa31A74Fe"
    "links": [
```

Comment (DragonCoin71) [inline blockchains/smartchain/assets/0x645660bD50F7229F1a975F1231c82EBAa31A74Fe/info.json:10]:
```suggestion
    "id": "0x645660bD50F7229F1a975F1231c82EBAa31A74Fe"
    "links": [
{
"name": "github",
"url": "https://github.com/DragonCoin71/"
},
{
"name": "twitter",
"url": "https://twitter.com/drondragoncoin?s=09"
},
{
"name": "telegram",
"url": https://t.me/DRAGON_DRON
},
{
"name": "whitepaper",
"url": "https://dragon-dron.com/index.php/whitepaper/"
},
{
"name": "source_code",
"url": "https://bscscan.com/address/0x645660bD50F7229F1a975F1231c82EBAa31A74Fe#code/"

}
]
}

```

Comment (DragonCoin71) [inline blockchains/smartchain/assets/0x645660bD50F7229F1a975F1231c82EBAa31A74Fe/info.json:10]:
```suggestion
    "id": "0x645660bD50F7229F1a975F1231c82EBAa31A74Fe"
    
    "links": [
        {
            "name": "github",
            "url": "https://github.com/DragonCoin71/"
        },
        {
            "name": "twitter",
            "url": "https://twitter.com/drondragoncoin?s=09"
        },
        {
            "name": "telegram",
            "url": "https://t.me/DRAGON_DRON/"
        },
        {
            "name": "whitepaper",
            "url": "https://dragon-dron.com/index.php/whitepaper/"
        },
        {
            "name": "source_code",
            "url": "https://bscscan.com/address/0x645660bD50F7229F1a975F1231c82EBAa31A74Fe#code /"
        }
    ]
}




```

Comment (DragonCoin71) [inline blockchains/smartchain/assets/0x645660bD50F7229F1a975F1231c82EBAa31A74Fe/info.json:10]:
```suggestion
    "id": "0x645660bD50F7229F1a975F1231c82EBAa31A74Fe"
    "links": [
        {
            "name": "github",
            "url": "https://github.com/DragonCoin71/"
        },
        {
            "name": "twitter",
            "url": "https://twitter.com/drondragoncoin?s=09"
        },
        {
            "name": "telegram",
            "url": "https://t.me/DRAGON_DRON/"
        },
        {
            "name": "whitepaper",
            "url": "https://dragon-dron.com/index.php/whitepaper/"
        },
        {
            "name": "source_code",
            "url": "https://bscscan.com/address/0x645660bD50F7229F1a975F1231c82EBAa31A74Fe#code /"
        }
    ]
}
```

Comment (DragonCoin71) [inline blockchains/smartchain/assets/0x645660bD50F7229F1a975F1231c82EBAa31A74Fe/info.json:10]:
```suggestion
    "id": "0x645660bD50F7229F1a975F1231c82EBAa31A74Fe"
    "links": [
        {
            "name": "github",
            "url": "https://github.com/DragonCoin71/"
        },
        {
            "name": "twitter",
            "url": "https://twitter.com/drondragoncoin?s=09"
        },
        {
            "name": "telegram",
            "url": "https://t.me/DRAGON_DRON/"
        },
        {
            "name": "whitepaper",
            "url": "https://dragon-dron.com/index.php/whitepaper/"
        },
        {
            "name": "source_code",
            "url": "https://bscscan.com/address/0x645660bD50F7229F1a975F1231c82EBAa31A74Fe#code /"
        }
    ]
}
```

**Comment by Cryptocool1** [comment]
correction needed:
 your background should be transparent.
 please change

**Comment by DragonCoin71** [comment]
logo

@Cryptocool1  I updated the new logo.png please check. thank you

**Comment by DragonCoin71** [comment]
logo png
 
I updated my logo. Please check. Thank you

**Comment by DragonCoin71** [comment]
@Cryptocool1 can you check the new logo file? we're in urgent case. Thanks.

**Comment by Cryptocool1** [comment]
still the same file

**Comment by DragonCoin71** [comment]
logo png

background is trasparent and Size: 256px by 256px , 57 kb
I fixed again. I hope it's right. I apologize for the problems.

**Comment by cryptomanz** [comment]
you just sent the comment, not changed

**Comment by DragonCoin71** [comment]
@cryptomanz Sorry how can i change? Should i open another PR? because i cant delete the logo and send the new one

**Comment by DragonCoin71** [comment]
Hello, im sorry for errors please help me i cant delete my logo and upload it. I sent the new one in comment.

**Comment by DragonCoin71** [comment]
Hello, i uploaded my logo but the bot said not allowed please help. Thank you so must.

**Comment by DragonCoin71** [comment]
Hello, i uploaded my logo but the bot said not allowed please help. Thank you so must.

**Comment by DragonCoin71** [comment]
Hello, i deleted and uploaded my logo but the bot said not allowed please help. Thank you so must.

**Comment by Cryptocool1** [comment]
incorrect naming, should be
blockchains/smartchain/assets/0x645660bD50F7229F1a975F1231c82EBAa31A74Fe/logo.png

**Comment by DragonCoin71** [comment]
sshot-11 (3)

I uploaded the logo with that name can you see in screenshot. I dont know what should i do now can you fix it for me? Thank you for your time

**Comment by DragonCoin71** [comment]
hi, please help me what should i do?? all of my file deleted? I should reopen the request? Please help. we're in urgent case. Thanks.

**Comment by DragonCoin71** [comment]
hi, please help me what should i do?? all of my file deleted? I should reopen the request? Please help. we're in urgent case. Thanks.

**Comment by DragonCoin71** [comment]
hi, please help me what should i do??I should reopen the request? Please help. we're in urgent case. Thanks.

**Comment by Cryptocool1** [comment]
should be logo.png, not logo.png.png

**Comment by Cryptocool1** [comment]
fixed

**Comment by Cryptocool1** [comment]
• What is the purpose of this token? How will it benefit Trust Wallet users?
 • How is the token being distributed? Are you conducting an airdrop?
 • Did you sold the tokens at an Initial Coin Offering (ICO) or a token presale?
 • Is the token listed on a centralized/decentralized exchange?
 • Is a Smart Contract audit completed? Where can we find this information?

**Comment by Cryptocool1** [comment]
info file should be named info.json exactly, not file.json

**Comment by Cryptocool1** [comment]
fixed

**Comment by DragonCoin71** [comment]
> • What is the purpose of this token? How will it benefit Trust Wallet users?

DragonCoin is a promising project for all the people of the world, Which means that we will maximize holders investment by DragonCoin project and we will spend part of the profits to use of Solar energy, So all our DRON investors and holders will be involved in our projects. And one of our plans is VDMT that all holders, 10% of their purchase are rewarded monthly, this encourages users to hold for the long term as they can use their rewarded tokens, so the longer they hold, the more tokens they will get for free.
Due to our various plans to hold token, our investors should be able to use a secure wallet, so we already promote Trust wallet to new investors that would be beneficial for both parties if our logo was updated.
We are frequently asked by members when our logo will be updated as the majority of our holders we believe use Trust Wallet. Having our logo displayed will also provide trust and confidence in our brand.
> • How is the token being distributed? Are you conducting an airdrop?

DragonCoin Token is being distributed by buying an selling through the pancakeswap exchange or Vodral exchange robot in Telegram. No, we did not do AirDrop, but we deposited Dragon Token as a gift to the active wallets. We also had a lottery for those who introduced new buyers and advertised on social media and the winners' gifts were deposited and all of them can be tracked.
> • Did you sold the tokens at an Initial Coin Offering (ICO) or a token presale?

No, we released the token on a public sale whilst marketing on telegram and twitter and instagram. No presale or ICO was conducted.
> • Is the token listed on a centralized/decentralized exchange?

Yes, our token is listed on pancakeswap as seen below. Conix Exchange Ambassador has sent us the listing form from the exchange and the initial steps for listing have been done.
https://exchange.pancakeswap.finance/#/swap?outputCurrency=0x645660bD50F7229F1a975F1231c82EBAa31A74Fe

> • Is a Smart Contract audit completed? Where can we find this information?

A smart contract audit is underway and has been requested to a company and they will send it in a few days.

**Comment by DragonCoin71** [comment]
> **PR Summary**
> Files OK, 2Files in PR: 2
> Token in PR: BEP20 0x645660bD50F7229F1a975F1231c82EBAa31A74Fe
> Symbol: **DRON** decimals: 8
> Links: [Logo](https://raw.githubusercontent.com/DragonCoin71/assets/br0707-125726/blockchains/smartchain/assets/0x645660bD50F7229F1a975F1231c82EBAa31A74Fe/logo.png) [Info](https://raw.githubusercontent.com/DragonCoin71/assets/br0707-125726/blockchains/smartchain/assets/0x645660bD50F7229F1a975F1231c82EBAa31A74Fe/info.json) [Website](https://dragon-dron.com/) [Explorer](https://bscscan.com/token/0x645660bD50F7229F1a975F1231c82EBAa31A74Fe)
> Token check error:
> ❌ Low token circulation: no. of holders is 4500, below limit of 5000
> ✓ Token type OK (BEP20); Contract/ID is OK; Type field OK; Symbol field OK; Decimals field OK; Description field OK; Website field OK; Explorer field OK; Status field OK; Id field OK; Logo image type is OK (image/png); Logo image size is OK (59.058 kB); Logo dimensions OK (256x256);
> 
> 


sshot-13

Hello, DragonCoin token holders are +5000 and transactions are +10,000. Please check. 
https://bscscan.com/token/0x645660bD50F7229F1a975F1231c82EBAa31A74Fe

Thank you so much.

**Comment by DragonCoin71** [comment]
hi,
can you check my holders and transfers? After that Is the logo visible? Our community is asking for it, we're in urgent case. thank you for your help.

**Comment by Cryptocool1** [comment]
awaiting for smartcontract audit

**Comment by DragonCoin71** [comment]
Hello, sorry for the delay! [https://www.dragon-dron.com/wp-content/uploads/2021/07/DragonCoin-Audit.pdf](url)

**Comment by DragonCoin71** [comment]
Hello, sorry for the delay!

https://www.dragon-dron.com/wp-content/uploads/2021/07/DragonCoin-Audit.pdf

**Comment by DragonCoin71** [comment]
@Cryptocool1 ,  @cryptomanz  Thank you very much for your efforts. You solve people's problems well and make the network and the trust wallet proud.

**Pull request merged**